product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -17,7 +17,8 @@ This mechanism has previously meant that if a transaction is pending commit and
 
 Additionally, handling the error could increase replication lag, due to a combination of the time taken to detect the deadlock, the time taken for the client to rollback its transaction, the time taken for indirect garbage collection of the already applied changes, and the time taken to redo the work. 
 
-This is where Parallel Apply’s deadlock mitigation, introduced in PGD 5.2, comes into play. For any transaction, Parallel Apply now looks at upcoming writes, and where there is a potential conflict, places the transaction on hold until the conflict has passed.
+This is where Parallel Apply’s deadlock mitigation, introduced in PGD 5.2, comes into play. For any transaction, Parallel Apply's receiver process look at the writers for transactions already scheduled to see if they are writing to a row that the current transaction wants to write. If it does, the receiver marks the row as needing to wait until the other transaction is committed before apply its change to the row.  
+This ensures that rows are written in the correct order.  
```

Comment:
My preference would be to treat PA like a black box. So I would drop receiver and writer terms.

"For any transaction, Parallel Apply looks at transactions already scheduled for a row that the current transaction want to write. If it finds one, the row is marked as needing to wait ..."